emoji/src/main/java/com/vanniktech/emoji/OnlyEmojisInputFilter.java

```diff
@@ -0,0 +1,33 @@
+package com.vanniktech.emoji;
+
+import android.text.InputFilter;
+import android.text.Spanned;
+import androidx.annotation.Nullable;
+
+public final class OnlyEmojisInputFilter implements InputFilter {
```

Comment:
This is also present in the other PR, did you mix these up?

Reply:
Seems like it - yeah.